### PR TITLE
Read secrets from an ENV file if mounted

### PR DESCRIPTION
**Pull request by bdunne**
This allows for a list of ENV vars to be read from a mounted file and exported at boot time.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -146,3 +147,5 @@ COPY --from=manifest /tmp/BUILD /opt/manageiq/manifest
 
 # Switch USER back to postgres
 USER 26
+
+CMD ["miq-run-postgresql"]
```
Comment (Fryguy) [inline Dockerfile:151]: Should this be entrypoint?  What happens if you set the `--entrypoint` to bash - will it still run this?

Comment (bdunne) [reply Dockerfile:151]: I tried an entrypoint script that would `exec container-entrypoint`, but the environment wasn't passed along.

Comment (bdunne) [reply Dockerfile:151]: > What happens if you set the --entrypoint to bash - will it still run this?

No, but it wouldn't run if it were an entrypoint script either.  What use case are you trying to solve?

File: Dockerfile [hunk 2]
```diff
@@ -134,6 +134,7 @@ RUN yum -y update postgresql-* && \
     yum clean all
 
 ADD container-assets/container-scripts /opt/manageiq/container-scripts/
+ADD container-assets/miq-run-postgresql /usr/bin/
```
Comment (Fryguy) [inline Dockerfile:137]: Super-super-minor, but I think we should move this into a separate "section", so it's not blended in with the upstream lines.  This way it's clear that it's ours and separate.

(i.e. move this down 3 lines into its own "paragraph" with a comment)

File: container-assets/miq-run-postgresql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+for file in /run/secrets/postgresql/*
```
Comment (Fryguy) [inline container-assets/miq-run-postgresql:3]: What happens if this directory doesn't exist or there are no files?

**Pull request merged**